packages/cli/src/commands/org/config/push.ts

```diff
@@ -0,0 +1,159 @@
+import {Command, flags} from '@oclif/command';
+import {cli} from 'cli-ux';
+import {ReadStream} from 'fs';
+import {dedent} from 'ts-dedent';
+import {cwd} from 'process';
+import {Config} from '../../../lib/config/config';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {Project} from '../../../lib/project/project';
+import {SnapshotFactory} from '../../../lib/snapshot/snapshotFactory';
+import {platformUrl} from '../../../lib/platform/environment';
+import {Snapshot} from '../../../lib/snapshot/snapshot';
+import {red, green, bold} from 'chalk';
+import {normalize} from 'path';
+
+export interface CustomFile extends ReadStream {
+  type?: string;
+}
+
+export default class Push extends Command {
+  public static description =
+    'Preview, validate and deploy your changes to the destination org';
+
+  public static flags = {
+    target: flags.string({
+      char: 't',
+      description:
+        'The unique identifier of the organization where to send the changes. If not specified, the organization you are connected to will be used.',
+      helpValue: 'destinationorganizationg7dg3gd',
+      required: false,
+    }),
+    projectPath: flags.string({
+      char: 'p',
+      description: 'The path to your Coveo project.',
+      helpValue: '/Users/Me/my-project',
+      default: cwd(),
+      required: false,
+    }),
+    deleteMissingResources: flags.boolean({
+      char: 'd',
+      description:
+        'Whether or not to delete resources missing in the destination',
+      default: false,
```

Comment:
Same comment as the other PR( https://github.com/coveo/cli/pull/258 ), should the default here be "the default of the API" ?

Reply:
Unfortunately, the flag is mandatory when interacting with the platform client
```
export interface DryRunOptions {
    deleteMissingResources: boolean;
}
export interface ApplyOptions {
    deleteMissingResources: boolean;
}
```